packages/cliqz-logo-database/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "cliqz-logo-database",
+  "version": "0.1.2",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "license": "MPL-2.0",
+  "dependencies": {
+    "tldts": "^5.0.3"
```

Comment:
for now it shouldn't be here

Reply:
it definitely must be there :-) otherwise it will be missing for consuming projects